src/main/java/com/foundrysoftware/minecraft/plugins/PluginTemplate.java

```diff
@@ -34,122 +34,135 @@
 import java.util.logging.Logger;
 
 public class PluginTemplate extends JavaPlugin {
-
+    private static final String ARROW_TYPE_TNT = "tnt";
     private final String PLUGIN_NAME = "PluginTemplate";
 
-    public void onEnable(){
-        getLogger().info("["+ PLUGIN_NAME +"] Started ");
+    public void onEnable() {
+        getLogger().info("[" + PLUGIN_NAME + "] Started ");
     }
 
-    public void onReload(){
-        getLogger().info("["+ PLUGIN_NAME +"] Server Reload");
+    public void onReload() {
+        getLogger().info("[" + PLUGIN_NAME + "] Server Reload");
     }
 
-    public void onDisable(){
-        getLogger().info("["+ PLUGIN_NAME +"] Server Stopping");
+    public void onDisable() {
+        getLogger().info("[" + PLUGIN_NAME + "] Server Stopping");
     }
 
     @Override
     public boolean onCommand(@NotNull CommandSender sender, @NotNull Command command, @NotNull String label, String[] args) {
-        if (label.equalsIgnoreCase("simple")){
+        if (label.equalsIgnoreCase("simple")) {
```

Comment:
Issue #1) All string literals should be constants aka "simple" and line 60. Please check every line of code and make them into constants